core/src/test/java/org/elasticsearch/discovery/DiscoveryWithServiceDisruptionsIT.java

```diff
@@ -490,7 +491,8 @@ public void testAckedIndexing() throws Exception {
                                 logger.trace("[{}] indexing id [{}] through node [{}] targeting shard [{}]", name, id, node, shard);
                                 IndexResponse response =
                                         client.prepareIndex("test", "type", id).setSource("{}").setTimeout(timeout).get(timeout);
-                                assertTrue("doc [" + id + "] should have been created", response.isCreated());
+                                assertThat("doc [" + id + "] should have been created", response.getOperation(),
```

Comment:
The official line wrap size is 140 characters, btw. You can wrap in whatever way you like, but checkstyle won't freak out at you until you get to 140 characters. There are like 1200 files that are grandfathered in, pre checkstyle, so it won't notice them. We're **slowly** removing files from that list. Basically when it doesn't make the diff horrible we'll include wrapping into another PR.